modules/ocl/cv_calibration_parser.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * cv_calibration_parser.cpp - parse fisheye calibration file
+ *
+ *  Copyright (c) 2016-2017 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Author: Junkai Wu <junkai.wu@intel.com>
+ */
+
+#include "cv_calibration_parser.h"
+
+namespace XCam {
+
+CalibrationParser::CalibrationParser()
+{
+}
+
+IntrinsicParameter
+CalibrationParser::getIntrinsicParam(char *file_str)
+{
+    parseIntrinsicParam(file_str);
+    return _intrinsicParam;
+}
+
+ExtrinsicParameter
+CalibrationParser::getExtrinsicParam(char *file_str)
+{
+    parseExtrinsicParam(file_str);
+    return _extrinsicParam;
+}
+
+void
+CalibrationParser::parseIntrinsicParam(char *file_str)
+{
+    char *line_str = NULL;
+    line_str = strtok(file_str, "\n");

```

Comment:
too specific. I think you better to use  "\r\n" for line tokens. since there's linux and dos line styles.
you can used a static const char *line_tokens ="\r\n"; instead.